### DISCUSSION TITLE
Windows and iPhoone

### DISCUSSION THREAD

**Discussion by NimuthuGanegoda** [kodjodevf/mangayomi#43] [Ideas]
Could you make this a .exe and a official app in the apple app store ?

**Comment by kodjodevf** [comment]
For an .exe I can do this, but for an official Apple app store I need to be supported for this to be possible.

**Reply by NimuthuGanegoda** [reply]
Ohh, I see then could you make a .exe, and How to take supported ?

**Reply by NimuthuGanegoda** [reply]
Can you make the the .exe in the next update ?

**Reply by NimuthuGanegoda** [reply]
Thanks for the .exe application
